From fb2de0e3639d98493f86571f20d888a4963921d9 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mikeioannina@gmail.com>
Date: Sat, 29 Aug 2015 13:15:02 +0300
Subject: [PATCH 001/201] Revert "SnapdragonCamera: Forbid volume key can take
 picture"

This reverts commit e73e717b8be1ba834161914af4debc61c1feb8be.

Change-Id: Iee719f34f6016f89ec0bb1c078bfa25c2f718436
---
 res/values/bool.xml                         | 1 -
 src/com/android/camera/CaptureModule.java   | 6 +-----
 src/com/android/camera/PhotoModule.java     | 6 +-----
 src/com/android/camera/util/CameraUtil.java | 4 ----
 4 files changed, 2 insertions(+), 15 deletions(-)

diff --git a/res/values/bool.xml b/res/values/bool.xml
index 2bb520bf6..4488d70bb 100644
--- a/res/values/bool.xml
+++ b/res/values/bool.xml
@@ -16,5 +16,4 @@
 <resources>
     <bool name="show_action_bar_title">false</bool>
     <bool name="force_count_down_sound">false</bool>
-    <bool name="volume_key_shutter_disable">false</bool>
 </resources>
diff --git a/src/com/android/camera/CaptureModule.java b/src/com/android/camera/CaptureModule.java
index 7b5403f8b..ca5050eab 100755
--- a/src/com/android/camera/CaptureModule.java
+++ b/src/com/android/camera/CaptureModule.java
@@ -3246,9 +3246,6 @@ public class CaptureModule implements CameraModule, PhotoController,
         switch (keyCode) {
             case KeyEvent.KEYCODE_VOLUME_UP:
             case KeyEvent.KEYCODE_VOLUME_DOWN:
-                if (CameraUtil.volumeKeyShutterDisable(mActivity)) {
-                    return false;
-                }
             case KeyEvent.KEYCODE_FOCUS:
                 if (mFirstTimeInitialized) {
                     if (event.getRepeatCount() == 0) {
@@ -3281,8 +3278,7 @@ public class CaptureModule implements CameraModule, PhotoController,
         switch (keyCode) {
             case KeyEvent.KEYCODE_VOLUME_UP:
             case KeyEvent.KEYCODE_VOLUME_DOWN:
-                if (mFirstTimeInitialized
-                        && !CameraUtil.volumeKeyShutterDisable(mActivity)) {
+                if (mFirstTimeInitialized) {
                     onShutterButtonClick();
                     return true;
                 }
diff --git a/src/com/android/camera/PhotoModule.java b/src/com/android/camera/PhotoModule.java
index f3c57cb71..cd78384f1 100644
--- a/src/com/android/camera/PhotoModule.java
+++ b/src/com/android/camera/PhotoModule.java
@@ -2824,9 +2824,6 @@ public class PhotoModule
         switch (keyCode) {
             case KeyEvent.KEYCODE_VOLUME_UP:
             case KeyEvent.KEYCODE_VOLUME_DOWN:
-                if (CameraUtil.volumeKeyShutterDisable(mActivity)) {
-                   return false;
-                }
             case KeyEvent.KEYCODE_FOCUS:
                 if (/*TODO: mActivity.isInCameraApp() &&*/ mFirstTimeInitialized) {
                     if (event.getRepeatCount() == 0) {
@@ -2902,8 +2899,7 @@ public class PhotoModule
         switch (keyCode) {
             case KeyEvent.KEYCODE_VOLUME_UP:
             case KeyEvent.KEYCODE_VOLUME_DOWN:
-                if (/*mActivity.isInCameraApp() && */ mFirstTimeInitialized
-                        && !CameraUtil.volumeKeyShutterDisable(mActivity)) {
+                if (/*mActivity.isInCameraApp() && */ mFirstTimeInitialized) {
                     onShutterButtonClick();
                     return true;
                 }
diff --git a/src/com/android/camera/util/CameraUtil.java b/src/com/android/camera/util/CameraUtil.java
index 34787e4d8..4dbe29132 100755
--- a/src/com/android/camera/util/CameraUtil.java
+++ b/src/com/android/camera/util/CameraUtil.java
@@ -1259,10 +1259,6 @@ public class CameraUtil {
         return ret;
     }
 
-    public static boolean volumeKeyShutterDisable(Context context) {
-        return context.getResources().getBoolean(R.bool.volume_key_shutter_disable);
-    }
-
     public static int determineRatio(int width, int height) {
         if (height != 0) {
             return determineRatio(((float) width) / height);
-- 
2.17.1

